docs/ROOTFS_DISCOVERY.md

```diff
@@ -0,0 +1,407 @@
+---
+title: Boot Components & Root File System Discovery
+category: Booting
+layout: default
+SPDX-License-Identifier: LGPL-2.1-or-later
+---
+
+# Boot Components & Root File System Discovery
+
+The recommended way to boot a [`systemd`](https://systemd.io/) based
+[UEFI](https://en.wikipedia.org/wiki/UEFI) system consists primarily of three
+components:
```

Comment:
I don't think this is a great way to phrase this. Can't we be more neutral and say e.g. "The systemd project provides the following components to boot a machine."

Also "systemd-based", needs a hyphen.